### PR TITLE
Review "Transactions and message processing"

### PR DESCRIPTION
**Pull request by weralabaj**
Resolves https://github.com/Particular/docs.particular.net/issues/1713.

### CODE REVIEW

**Comment by weralabaj** [comment]
I thought about splitting this article and re-organizing content, but I think that might easily get out of control and become a bigger effort, so I'd rather leave it for https://github.com/Particular/PlatformDevelopment/issues/900 and just stick to the "good enough" for now.

File: nservicebus/operations/transactions-message-processing.md [hunk 1]
```diff
@@ -1,86 +1,71 @@
...
 
 ## Distributed Transaction Coordinator
 
-In Windows, there is an OS-level service called the DTC that manages transactions needing to span multiple resources, like queues and databases. This service isn't always configured correctly and may require troubleshooting. Download a tool called [DTCPing](https://www.microsoft.com/en-us/download/details.aspx?id=2868) to help discover if one machine can access a remote machine over the DTC. The tool looks like this.
-
-![this is what the initial DTCPing window to look like.](dtcping.png "this is what the initial DTCPing window to look like.")
+In Windows, the Distributed Transaction Coordinator (DTC) is an OS-level service which manages transactions that span across multiple resources, e.g. queues and databases.
 
-If an error referring to the RPC Endpoint Mapper occurs, at the command prompt, run `dcomcnfg`. Note the Component Services screen below.
+The easiest way to configure DTC for NServiceBus is to run the [PlatformInstaller](http://docs.particular.net/platform/installer/) for NServiceBus, or to use the dedicated [Powershell commandlets](/nservicebus/operations/management-using-powershell.md).
```
Comment (SimonCropp) [inline nservicebus/operations/transactions-message-processing.md]: Should not fully qualify the pi link

**Comment by gbiellem** [comment]
I'd suggest we avoid having our own doco on troubleshooting MSDTC with DTCPing and instead link to some existing MS doco.  This one is better than what we have - https://blogs.msdn.microsoft.com/puneetgupta/2008/11/12/troubleshooting-msdtc-issues-with-the-dtcping-tool/

**Comment by weralabaj** [comment]
@gbiellem I think the same would make sense for http://docs.particular.net/nservicebus/operations/warning-for-cid-values? And probably to make it a section in the document I've reviewed in this PR.
What do you think?

**Comment by weralabaj** [comment]
@gbiellem @SimonCropp see now

**Comment by gbiellem** [comment]
@weralabaj  I like it.

**Pull request merged**